maintenance/maintenance.html

```diff
@@ -114,37 +114,34 @@ <h4 class="h3-heading mb-4">Let’s protect the world’s largest resource for f
             <div class="row">
-              <div class="col-lg-4">
+              <div class="col-lg-6 col-xl-5">
                 <h5 class="h5-heading">More about us</h5>
                 <ul class="d-flex">
                   <li class="mr-2"><a class="twitter-glyph small d-inline-block" href="https://twitter.com/mozilla"><span class="sr-only">Twitter</span></a></li>
                   <li class="mr-2"><a class="instagram-glyph small d-inline-block" href="https://www.instagram.com/mozilla/"><span class="sr-only">Instagram</span></a></li>
                   <li class="mr-2"><a class="github-glyph small d-inline-block" href="https://www.github.com/mozilla/foundation.mozilla.org/"><span class="sr-only">Github</span></a></li>
                 </ul>
               </div>
-              <div class="col-lg-8">
+              <div class="col-lg-6 col-xl-7">
                 <ul class="link-list list-unstyled mb-0">
                   <li class="mb-2"><a id="donate-footer-btn" href="https://donate.mozilla.org/" target="_blank" rel="noopener noreferrer" class="dark-theme">Donate</a></li>
+                  <li class="mb-2"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
                   <li class="mb-2"><a href="https://www.mozilla.org/about/legal/terms/mozilla/" class="dark-theme">Legal</a></li>
-                  <li class="mb-2"><a href="https://creativecommons.org/licenses/by/4.0" class="dark-theme">License</a></li>
                   <li class="mb-2"><a href="https://www.mozilla.org/about/governance/policies/participation/" class="dark-theme">Participation Guidelines</a></li>
+                  <li><a href="https://foundation.mozilla.org/en/press-center/" class="dark-theme">Press Center</a></li>
                   <li class="mb-2"><a href="https://mozilla.org/en-US/privacy/websites/" class="dark-theme">Privacy</a></li>
-                  <li class="mb-2"><a href="https://www.mozilla.org/privacy/websites/#cookies" class="dark-theme">Cookies</a></li>
                 </ul>
               </div>
             </div>
-            <hr class="mt-3 mt-md-5" />
             <div class="row">
               <div class="col-12">
-                <p class="dark-theme">Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good.</p>
-                <p class="dark-theme mb-0">Visit us at <a href="https://mozilla.org">mozilla.org</a></p>
+                <hr class="mt-5 mb-3" />
+                <p class="dark-theme body-small mb-0">Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://foundation.mozilla.org">foundation.mozilla.org</a>. Most content available under a <a href="https://foundation.mozilla.org/about/website-licensing/">Creative Commons license</a>.</p>

```

Comment:
I wonder if we should `{% blocktrans %}` this, since we're touching it now.

Reply:
`maintenance.html` is static page so I think Django tags won't work here.

Reply:
oh right, that's a good point. I said nothing! =D